packages/myst-migrate/src/migrations.ts

```diff
@@ -0,0 +1,4 @@
+import type { Migration } from './types.js';
+import * as v1 from './v1_footnotes.js';
+
+export const MIGRATIONS: Migration[] = [v1];
```

Comment:
This types each module for an upgrade/downgrade/description, and we then just put it in an array.